### DISCUSSION TITLE
Mode expert > lookup dns

### DISCUSSION THREAD

**Discussion by adriens** [lschaeffer313/domaine-nc-mobile#108] [Polls]
" :grey_question: Contexte

Actuellement il n'est pas possible de faire un lookup en live sur le nom de domaine. cela est possible avec la librairie [dns_client ](https://pub.dev/packages/dns_client)

**Comment by tamere-allo-peter** [comment]
Oui mais... les requêtes vont alors certainement utiliser les serveurs DNS mis à disposition par la connexion active du téléphone. S'il s'agit d'une connexion 4G, les serveurs DNS utilisés sont ceux de la DT, et ils font office de cache. Il est donc possible que les informations récupérées par des requêtes DNS ne soient pas parfaitement à jour (et c'est normal).

**Reply by adriens** [reply]
WOW, top ton retour @tamere-allo-peter , c'est supe rintéressant, il faudra sans doute mentionner cela sur cet écran du coup, mais ça permet d'avoir une connaissance super fine sur les expériences mobiles du coup @lschaeffer313 :open_mouth:

**Reply by lschaeffer313** [reply]
Merci pour les explications @tamere-allo-peter.
Il faudra réfléchir à comment l'afficher sur un écran mobile 🤔

**Reply by tamere-allo-peter** [reply]
Bin un truc genre "ATTENTION : il est possible que les informations DNS affichées ne soient pas parfaitement à jour..." dans un scrolling horizontal en haut de l'écran, sur un fond étoilé qui défile vers le haut, et avec de temps en temps un vaisseau qui traverse et qu'on peut shooter avec le bouton On/Off de son téléphone.

**Reply by adriens** [reply]
:open_mouth:  WTF @tamere-allo-peter mais c'est quoi ta marque de camomille :grey_question: :grey_exclamation: ... JE VEUX LA MEME :rofl:

**Reply by lschaeffer313** [reply]
... avec le faucon Millenium qui est le vaisseau du joueur 🤣

**Reply by adriens** [reply]
![image](https://user-images.githubusercontent.com/5235127/210457344-f71b908b-0ab9-4a19-aaf7-25eba0ae8fea.png)

**Reply by tamere-allo-peter** [reply]
@adriens : Rhum agricole aux plantes endémiques...

![x](https://user-images.githubusercontent.com/85658105/210457690-efb0e43c-3d7e-4223-8d44-a25ed346f344.jpg)
